pkg/resourceinterpreter/default/native/reflectstatus.go

```diff
@@ -74,7 +75,20 @@ func reflectDeploymentStatus(object *unstructured.Unstructured) (*runtime.RawExt
 		AvailableReplicas:   deploymentStatus.AvailableReplicas,
 		UnavailableReplicas: deploymentStatus.UnavailableReplicas,
 	}
-	return helper.BuildStatusRawExtension(grabStatus)
+
+	grabStatusRaw, err := helper.BuildStatusRawExtension(grabStatus)
+	if err != nil {
+		return nil, err
+	}
+
+	// if status is empty struct, it actually means status haven't been collected from member cluster.
+	if bytes.Equal(grabStatusRaw.Raw, []byte("{}")) {
+		klog.Errorf("Empty status grabed rom %s(%s/%s) which should been non-empty field.",
+			object.GetKind(), object.GetNamespace(), object.GetName())
```

Comment:
Is it worth logging an `error`?
In addition, there are 3 typos there, `grabed`, `rom`, `been`.

Reply:
beacuse above not exist log is `errorf`, it should keep the same, so should we update the two into `info`?

https://github.com/karmada-io/karmada/blob/db15f17b8ecfa0e606579a175fc9371b45c0fd9b/pkg/resourceinterpreter/default/native/reflectstatus.go#L59-L63

Reply:
sorry for typo error, it will be fix as:

```suggestion
		klog.Errorf("Empty status grabbed from %s(%s/%s) which should be non-empty field.",
			object.GetKind(), object.GetNamespace(), object.GetName())
```